app/models/game.rb

```diff
@@ -6,4 +6,43 @@ class Game < ActiveRecord::Base
 
   delegate :pawns, :rooks, :knights, :bishops, :queens, :kings, to: :pieces
   scope :available, -> { where('white_player_id IS NULL OR black_player_id IS NULL') }
+
+  def populate_board!
+    #- white - 16
+    # Pawn - 8
+    (0..7).each do |i|
+      pieces.create(type: 'Pawn', x_position: i, y_position: 0, color: 0)
```

Comment:
because color is an enum, you can do `color: 'black'`. Which is more readable than `0`. 😄